x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/helpers.ts

```diff
@@ -109,9 +109,14 @@ export const streamGraph = async ({
             }
           }
         }
+      } else if (event.event === 'on_llm_end') {
```

Comment:
@jacoblee93 I'm getting 2 types of events here, ones that come from `GENERATE_CHAT_TITLE_NODE` and ones that come from `PERSIST_CONVERSATION_CHANGES_NODE`. If I do not filter between the events, I get the result of the chat title step as a part of the message stream output.

 The only param I can see to differentiate the event types is the title events have:
 ```
  "tags": [
    "seq:step:2"
  ]
```
 and the message events have: 
 ```
  "tags": [
    "seq:step:3"
  ]
 ```

If I filter for `event.tags.includes('seq:step:3')`, I get only the message output as desired. Is filtering by these tags a safe way to accomplish this? Is there a way to keep events from `GENERATE_CHAT_TITLE_NODE` from making it to this stream? Is there a way to add a static tag to the event incase another step is added?